src/Cache.php

```diff
@@ -67,9 +67,16 @@ public function psr(): \Psr\SimpleCache\CacheInterface
         return $this->psr;
     }
 
+    /**
+     * @param mixed $key
+     * @psalm-param callable(\Psr\SimpleCache\CacheInterface): mixed $callable
+     *
+     * @param DateInterval|int|null $ttl
+     */
     public function getOrSet($key, callable $callable, $ttl = null, Dependency $dependency = null, float $beta = 1.0)
```

Comment:
That's interface implementation. I guess annotations should be added there and not here.